testsuite/features/secondary/min_ansible_control_node.feature

```diff
@@ -12,31 +15,62 @@ Feature: Operate an Ansible control node in a normal minion
      Then I should see a "Ansible Control Node type has been applied." text
 
   Scenario: Apply highstate and check that Ansible is installed
-    Given I am on the Systems overview page of this "sle_minion"
-    When I follow "States" in the content area
-    And I click on "Apply Highstate"
-    And I wait until event "Apply highstate scheduled by admin" is completed
-    Then "ansible" should be installed on "sle_minion"
+     Given I am on the Systems overview page of this "sle_minion"
+     When I follow "States" in the content area
+     And I click on "Apply Highstate"
+     And I wait until event "Apply highstate scheduled by admin" is completed
+     Then "ansible" should be installed on "sle_minion"
 
   Scenario: The Ansible tab appears in the system overview page
      Given I am on the Systems overview page of this "sle_minion"
      When I follow "Ansible" in the content area
      Then I should see a "Ansible Control Node Configuration" text
 
-  Scenario: Pre-requisite: Deploy test playbooks and inventory file
-     Given I deploy testing playbooks and inventory files to "sle_minion" 
+  Scenario: Configure some inventory and playbooks path
+     Given I am on the Systems overview page of this "sle_minion"
+     When I follow "Ansible" in the content area
+     Then I should see a "Ansible Control Node Configuration" text
+     And I enter "/srv/playbooks/" as "new_playbook_path_input"
+     And I click on button with id "new_playbook_path_save"
+     And I enter "/srv/playbooks/example_playbook2_orion_dummy/hosts" as "new_inventory_path_input"
+     And I click on button with id "new_inventory_path_save"
+
+  Scenario: Display inventories
+     Given I am on the Systems overview page of this "sle_minion"
+     When I follow "Ansible" in the content area
+     And I follow "Inventories" in the content area
+     And I wait until I see "/srv/playbooks/example_playbook2_orion_dummy/hosts" text
+     Then I click on "/srv/playbooks/example_playbook2_orion_dummy/hosts" text in Ansible paths
```

Comment:
Try to use this step instead:
`I click on "/srv/playbooks/example_playbook2_orion_dummy/hosts" in element "ansible-path-content"`

Not sure if it will work, due to the extra filter 'h6' you added, but it's worth trying it, and safe an additional step.

Reply:
As mentioned above, that didn't work for me